phoenix-core/src/it/java/org/apache/phoenix/end2end/QueryMoreIT.java

```diff
@@ -511,4 +513,122 @@ private void upsertRows(PhoenixConnection conn, String fullTableName) throws SQL
             stmt.execute();
         }
     }
+
+    @Test public void testRVCWithDescAndAscendingPK() throws Exception {
```

Comment:
nit:annotation should be in separate line.

Reply:
Sure